#pragma once

#include <QtWidgets/QMainWindow>
#include "ui_systemhelp.h"

class systemhelp : public QMainWindow
{
    Q_OBJECT

public:
    systemhelp(QWidget *parent = Q_NULLPTR);
	public slots:
	void OnBtn1Clicked();
	void OnBtn2Clicked();
	void OnBtn3Clicked();
	void OnBtn4Clicked();
	void OnBtn5Clicked();
	void OnBtn6Clicked();
	void OnBtn7Clicked();
	void OnBtn8Clicked();
	void OnBtn9Clicked();
	void OnBtn10Clicked();
	void OnBtn11Clicked();
	void OnBtn12Clicked();
	void OnBtn13Clicked();
	void OnBtn14Clicked();
	void OnBtn15Clicked();
	void OnBtn16Clicked();
	void OnBtn17Clicked();
	void OnBtn18Clicked();
	void OnBtn19Clicked();
	void OnBtn20Clicked();
	void OnBtn21Clicked();
private:
    Ui::systemhelpClass ui;
};
